webapp/src/components/EventStatus.js

```diff
@@ -285,6 +285,20 @@ class EventStatus extends Component {
     }
   };
 
+  applicationOpeningStatus = (event) => {
+    return {
+      title: this.props.t("Applications Opening Soon"),
+      titleClass: "text-warning",
+      longText:
+        this.props.t("Applications for") +
+        ` ${event.name} ` +
+        this.props.t("will open on") +
+        ` ${event.application_open_date}. ` +
+        this.props.t("Please check back later to apply."),
```

Comment:
For this kind of long text, to ensure that it gets translated correctly, you should use a translation block with placeholders to subsitute in the variables. See here for an example: https://github.com/deep-learning-indaba/Baobab/blob/8fb7327c8976bcd54d278804a321c7609811cca5/webapp/src/pages/references/components/ReferenceComponent.js#L106

Reply:
(I know the rest of the examples in this file are wrong 🙈  if you could fix those too that would be highly apreciated! Thank you :) )